test.js

```diff
@@ -40,8 +43,9 @@ test('StuckTransactionsCanceller', async t => {
           throw new Error('Should not be called')
         }
       },
-      log: () => {
+      log: str => {
         // TODO: Test logs
```

Comment:
Are the comments `// TODO: Test logs` still relevant? Can we remove them now?

Reply:
They are still relevant since we're not yet testing logs